### DISCUSSION TITLE
或许可以修改一下关于 AVATAR_PROXY 和 GRAVATAR_STR 的说明

### DISCUSSION THREAD

**Discussion by heganyuliang** [walinejs/waline#1026] [Ideas]
因为 workers.dev 被污染，今天修改头像地址时试错了太多次。看目前的备注以及 #464 的说法，第一反应是把`AVATAR_PROXY`更改为可访问地址，但这样无法获取到正确头像。最后成功的操作是把它设为`false`，而把`GRAVATAR_STR`改为类似`https://cravatar.cn/avatar/{{mail|md5}}`这样的形式。建议修改一下文档里关于这两项的描述，应该能帮助有同样需要的人节省大量时间。

**Comment by ee-blog** [comment]
有用，谢

---
Useful, thanks
